jenkins_job_wrecker/modules/publishers.py

```diff
@@ -78,30 +78,104 @@ def fingerprinter(top, parent):
             ext_email['reply-to'] = element.text
         elif element.tag == 'contentType':
             ext_email['content-type'] = element.text
+        elif element.tag == 'compressBuildLog':
+            ext_email['compress-log'] = get_bool(element.text)
         elif element.tag == 'defaultSubject':
             ext_email['subject'] = element.text
         elif element.tag == 'defaultContent':
             ext_email['body'] = element.text
         elif element.tag in ['attachBuildLog', 'compressBuildLog']:
             ext_email['attach-build-log'] = (element.text == 'true')
         elif element.tag == 'attachmentsPattern':
-            ext_email['attachment'] = element.text
-        elif element.tag in ['saveOutput', 'disabled']:
-            pass
+            ext_email['attachments'] = element.text
+        elif element.tag == 'saveOutput':
+            ext_email['save-output'] = get_bool(element.text)
         elif element.tag == 'preBuild':
             ext_email['pre-build'] = (element.text == 'true')
         elif element.tag == 'presendScript':
             ext_email['presend-script'] = element.text
+        elif element.tag == 'postsendScript':
+            ext_email['postsend-script'] = element.text
         elif element.tag == 'sendTo':
             ext_email['send-to'] = element.text
         elif element.tag == 'configuredTriggers':
-            print("IGNORED configuredTriggers in email-ext")
+            trigger_prefix = "hudson.plugins.emailext.plugins.trigger."
+            trigger_dict = {
+                "AlwaysTrigger": "always",
+                "UnstableTrigger": "unstable",
+                "FirstFailureTrigger": "first-failure",
+                "FirstUnstableTrigger": "first-unstable",
+                "NotBuiltTrigger": "not-built",
+                "AbortedTrigger": "aborted",
+                "RegressionTrigger": "regression",
+                "FailureTrigger": "failure",
+                "SecondFailureTrigger": "second-failure",
+                "ImprovementTrigger": "improvement",
+                "StillFailingTrigger": "still-failing",
+                "SuccessTrigger": "success",
+                "FixedTrigger": "fixed",
+                "FixedUnhealthyTrigger": "fixed-unhealthy",
+                "StillUnstableTrigger": "still-unstable",
+                "PreBuildTrigger": "pre-build"
+            }
+            trigger_result = {}  # To be able to print all on the output yaml
+            for key in trigger_dict.keys():
+                trigger_result[key] = False
+            # JJB implementation only uses the default content-type/subject/message
+            # and uses those in the trigger elements. Does not take information
+            # from trigger element's content. However, sendTo must be taken from
+            # one of the triggers, in this case, first trigger will be used in order
+            # to retrieve sendTo information.
+            is_send_to_received = False
```

Comment:
Are you sure this is what people would expect? I would expect to see a raw XML dump if my configuration couldn't be converted 1:1...

Reply:
Honestly no, that is why I wanted to ask your opinion on this. We will either have to change JJB implementation to cover all kinds of possibilities (which will need to apply a pattern to handle because it's huge). How do you think we should approach to this?

Reply:
So maybe you should just check `is_send_to_received` as below in line 140 and just raise NotImplementedError 'send-to' has already been filled in. This should cover the basic scenarios with conversion to YAML and the advanced scenarios with raw XML dump... Is that correct?